src/lib/viewers/doc/DocControls.tsx

```diff
@@ -40,12 +40,16 @@ export default function DocControls({
     pageCount,
     pageNumber,
     scale,
+    thumbnailsSidebarIsOpen,
```

Comment:
Let's use `isThumbnailsOpen` to stay consistent with the callback naming?

Reply:
updated!